02-microarray/pathway-analysis_microarray_03_gsea.Rmd

```diff
@@ -67,7 +67,7 @@ In the same place you put this `.Rmd` file, you should now have three new empty
 ## Obtain the gene set for this example
 
 In this example, we are using differential expression results table we obtained from an [example analysis of zebrafish samples overexpressing human CREB experiment](https://alexslemonade.github.io/refinebio-examples/02-microarray/differential-expression_microarray_01_2-groups.html) using [`limma`](https://bioconductor.org/packages/release/bioc/html/limma.html) [@Ritchie2015].
-The table contains Ensembl gene IDs, log fold-changes, and adjusted p-values (FDR in this case).
+The table contains Ensembl gene IDs, t-statistic values, and adjusted p-values (FDR in this case).
```

Comment:
Did we change the output table? What does the output table actually have? I thought both were included?

Reply:
Yes both are included, however there are other values that are included in the table that are not listed here (i.e. `B`, `AveExpr`, and `P.Value`).

I figured if we were not mentioning all of the columns in the table, we would mention only those that we would be using in the notebook. However, I can adjust this to mention all 8 columns that we have in the output table.

Reply:
In that case, let's do a slight wording change:

```suggestion
The table contains summary statistics including Ensembl gene IDs, t-statistic values, and adjusted p-values (FDR in this case).
```